### ISSUE TITLE
Can't open game on Windows

### ISSUE THREAD

**Issue by Pyro55** [FallowWing/Interphore#18]
![image](https://user-images.githubusercontent.com/39171844/39892846-cc2f709a-5456-11e8-97a3-e634dfb93e4f.png)
![image](https://user-images.githubusercontent.com/39171844/39892861-d8d38106-5456-11e8-9d36-dfea3085e803.png)
![image](https://user-images.githubusercontent.com/39171844/39892869-e1ce458e-5456-11e8-9374-cea82e5bd975.png)
I receive these errors instea

**Comment by FallowWing** [comment]
I statically linked the CRT and updated the game. [Does it still happen?](http://paraphore.com/interphore/interphore.zip)

**Comment by Pyro55** [comment]
Yes it keeps happening. Also when I downloaded the APK for Android,
Interphore kept crashing every time I tried to open it. (Also Parapore has
some issues where big blocks of text that extend past the next button can't
be scrolled to read so I can only read the bit that is not cut off)

**Comment by FallowWing** [comment]
I can't imagine why you'd get that same error, maybe you're using a cached version of the zip file?

**Comment by Pyro55** [comment]
Every time I've downloaded it on my phone or computer it crashed as soon as
I tried to open it. It's possible, but I've been downloading the version
from the website. Is that not the most updated version?

**Comment by FallowWing** [comment]
Here's the most reason version of Interphore for [Windows](https://www.dropbox.com/s/c663mbxiywbw030/Interphore.zip?dl=0), and [Android](https://www.dropbox.com/s/l7sxyyh7xloi2ju/Interphore.apk?dl=0).

Do these versions exhibit these issues?
